test/integration/jenkins/serverspec/rbenv_spec.rb

```diff
@@ -0,0 +1,98 @@
+require "spec_helper"
+
+describe "jenkins-ci::rbenv" do
+  describe "installs rbenv" do
+    describe file('/opt/rbenv') do
+      it { should be_directory }
+      it { should be_readable.by_user('jenkins') }
+      it { should be_writable.by_user('rbenv') }
+      it { should be_executable.by_user('jenkins') }
+      it { should be_owned_by 'rbenv' }
+      it { should be_grouped_into 'rbenv' }
+    end
+    print_env = command('sudo -u jenkins -i printenv').stdout.chomp
+    it "has RBENV_ROOT set to /opt/rbenv" do
+      expect(print_env).to match %r{RBENV_ROOT=/opt/rbenv}
+    end
+    which_rbenv = command('sudo -u jenkins -i which rbenv').stdout.chomp
+    it "has rbenv in search path" do
+      expect(which_rbenv).to match %r{/opt/rbenv/bin/rbenv}
+    end
+    describe file(which_rbenv) do
+      it { should be_linked_to '../libexec/rbenv' }
+      it { should be_owned_by 'rbenv' }
+      it { should be_grouped_into 'rbenv' }
+    end
+    describe file('/opt/rbenv/libexec/rbenv') do
+      it { should be_file }
+      it { should be_mode 755 }
+      it { should be_owned_by 'rbenv' }
+      it { should be_grouped_into 'rbenv' }
+    end
+  end
+
+  %w(
+    1.9.3-p448
+    2.1.2
+  ).each do |ruby|
```

Comment:
Thank you. I try.
It stopped me from being able to run the integration tests early on in the refactor. I built out the tests to find the problems, and I will need them to expand the supported pantheon of OSes.